src/i18n/locales/es.json

```diff
@@ -293,6 +293,20 @@
     "sha_384": "SHA-384",
     "sha_512": "SHA-512"
   },
+  "language": {
+    "de": "Alemán",
+    "en": "Inglés",
+    "es": "Español",
+    "fr": "Francés",
+    "hi": "hindi",
```

Comment:
The text is all correct, I would just put the first char as uppercase on all of them

Reply:
Done.